pom.xml

```diff
@@ -75,8 +75,7 @@
     <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
     <jackson.groupId>com.fasterxml.jackson.core</jackson.groupId>
     <jackson.package>com.fasterxml.jackson</jackson.package>
-    <jackson.version>2.9.9</jackson.version>
-    <jackson-databind.version>2.9.9.3</jackson-databind.version>
```

Comment:
Once, we had to separate the databind version because it was released separately. I would suggest keeping them separated and use the same version for both.

Reply:
ok, done now.